src/braket/jobs/config.py

```diff
@@ -29,6 +29,7 @@ class InstanceConfig:
 
```

Comment:
Do we want to regionalize this config? In the event we have an issue with m5.Ls in one region, this would allow a quick change without affecting other regions.

Reply:
Instead of including each region, we can have a default value that can be overridden for regions. This way, we don't explicitly need to write out each region's value and can change only when it is needed to be specified outside the default.

Reply:
Resolved offline: conclusion was that these configs are typically used for a single job, which only exists in one region